frappe/commands/__init__.py

```diff
@@ -112,7 +112,7 @@ def get_commands():
 	from .translate import commands as translate_commands
 	from .utils import commands as utils_commands
 
-	clickable_link = "\x1b]8;;https://frappeframework.com/docs\afrappeframework.com\x1b]8;;\a"
```

Comment:
Removed ansi escape sequence for URL, not yet standardized across most popular terminals. :see_no_evil: